Cargo.toml

```diff
@@ -31,5 +31,5 @@ futures = "0.3.17"
 nix = "0.23.0"
 rand = "0.8.3"
 faccess = "0.2.3"
-tokio = { version = "1.5.0", features = ["full"] }
+tokio = { version = "1.14.0", features = ["full"] }
```

Comment:
should probably not bump the **library** min ver.
`Cargo.lock` is for the binary